server/start.go

```diff
@@ -630,65 +627,6 @@ func getCtx(svrCtx *server.Context, block bool) (*errgroup.Group, context.Contex
 	return g, ctx
 }
 
-// startGrpcServer starts a gRPC server based on the provided configuration.
-func startGrpcServer(
```

Comment:
why is it ok to remove all of this?

Reply:
since we can replace with sdk [StartGrpcServer](https://github.com/cosmos/cosmos-sdk/blob/main/server/start.go#L473)?